frontend/src/components/common/formQuestions/AddQuestionModal/index.tsx

```diff
@@ -35,6 +35,7 @@ type AddQuestionModalProps = {
   ) => void;
   questionToBeEdited?: CreateFormQuestion;
   editing?: boolean;
+  isTemplatePage?: boolean;
```

Comment:
nit: I'd just name it the fuller name `isFormTemplatePage` for clarity, makes the context extra clear